configs/xcache-redir/config.d/03-redir-tunning.cfg

```diff
@@ -0,0 +1,10 @@
+# Retries should actually be 0, but 1 is minimum                                              
+# cms.dfs limit 0 lookup distrib mdhold 0 redirect immed retries 1                                                                                
+# cms.sched maxretries 3 nomultisrc affinity strict                                                                                                                         
+cms.sched maxretries 0 nomultisrc space 100 fuzz 5
+# cms.sched maxretries 1@cmsxrootd.fnal.gov:1094 nomultisrc                                                                                                           
+cms.fxhold noloc 15m 4h
```

Comment:
What's with all of the commented out lines here? And the comment says that 1 is the minimum retry but you set it to zero here. Does the comment mean that `maxretries 0` actually means that it will retry once?

Reply:
Yea I changed it this was what Matevz had left at UCSD. Basically we need 0 retries otherwise same file can end up in multiple nodes of the same cache. Which ends up wasting lots of space

Reply:
Could you remove the extra commented lines and update the comment to reflect that we set 0 retries to prevent file duplication across caches?

Reply:
Yep done